zaza/openstack/charm_tests/ceph/tests.py

```diff
@@ -776,6 +775,121 @@ def test_cinder_ceph_restrict_pool_setup(self):
             raise zaza_exceptions.CephPoolNotConfigured(msg)
 
 
+class CephMonReweightActionsTest(unittest.TestCase):
+    """Test ceph mon reweight actions."""
+
+    @classmethod
+    def setUpClass(cls):
+        """Run class setup for running tests."""
+        super(CephMonReweightActionsTest, cls).setUpClass()
+
+    def test_reweight_by_utilization_no_threshold(self):
+        """Test the reweight-by-utilization (no threshold) action.
+
+        The reweight-by-utilization action is checked for completion.
+        """
+        logging.info('Checking reweight-by-utilization action...')
+        unit_name = 'ceph-mon/0'
+
+        action_obj = zaza_model.run_action(
+            unit_name=unit_name,
+            action_name='reweight-by-utilization',
+        )
+        self.assertEqual('completed', action_obj.status)
+        zaza_model.block_until_unit_wl_status(
+            unit_name,
+            'active'
+        )
+        logging.debug('OK')
+
+    def test_reweight_by_utilization(self):
+        """Test the reweight-by-utilization action.
+
+        The reweight-by-utilization action is checked for completion.
+        """
+        logging.info('Checking reweight-by-utilization action...')
+        unit_name = 'ceph-mon/0'
+
+        reweight_params = {
+            'threshold': 120
```

Comment:
"Percentage at which OSDs should have weights adjusted downward"

120 seems like and odd percentage to test with and is is actually valid?